torchacc/utils/patch.py

```diff
@@ -92,3 +93,38 @@ def update_causal_mask(
             return None
 
         LlamaModel._update_causal_mask = update_causal_mask
+
+def patch_qwen():
+    '''
+    Modify the calculation of `rotary_seq_len` in `Qwen2FlashAttention2.forward` to avoid xla graph be executed
+    '''
+    import inspect
+    import transformers
+    from .logger import logger
+    from packaging import version
+    if version.parse(transformers.__version__) >= version.parse("4.37.0"):
```

Comment:
Have we checked that the latest version of transformers (4.46.3) can be patched properly?

Reply:
When transformers's version >= v4.45.0, It remove the `tensor.item` call, so it seems this patch doesn't be needed in the future